src/ReactiveDomain.UI/ReactiveDomain.UI.csproj

```diff
@@ -1,26 +1,25 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="../ci.build.imports" />
   <PropertyGroup>
-    <TargetFrameworks>$(LibTargetFrameworks)</TargetFrameworks>
+    <TargetFrameworks>net472</TargetFrameworks>
```

Comment:
This seems like a bad change.

Reply:
this only works on that framework, wpf isn't supported in net core/net standard

Reply:
got it.